main/main.go

```diff
@@ -96,7 +96,12 @@ func main() {
 		panic(err)
 	}
 
-	network, responseChans, err := peers.NewNetwork(logger, registerer, cfg.NetworkID, sourceSubnetIDs, sourceBlockchainIDs, cfg.PChainAPIURL)
+	// Unless the log level is debug, set the network log level to error
+	networkLogLevel := logging.Error
+	if logLevel == logging.Debug {
+		networkLogLevel = logging.Debug
+	}
```

Comment:
Why wouldn't we use the same log level in all cases here? (Could you add as a comment either way)

Reply:
The p2p network generates logs that aren't necessarily pertinent to the core functionality of the relayer. In order to reduce the amount of log noise, we provide different log levels for each. Added a comment explaining this